src/arm/radxa_cm3_io.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2022 Radxa Computer Co.,Ltd
+ * Author: Nascs <nascs@vamrs.com>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <mraa/common.h>
+#include <stdarg.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+
+#include "arm/radxa_cm3_io.h"
+#include "common.h"
+
+#define DT_BASE "/proc/device-tree"
+
+#define PLATFORM_NAME_RADXA_CM3_IO "Radxa CM3 IO"
+#define PLATFORM_NAME_RADXA_CM3_IO_2 "Radxa CM3 IO"
```

Comment:
This and the following usage is for product whose name is changed in device tree. As there is only one name currently for CM3 IO, they should be removed.